trio/tests/test_exports.py

```diff
@@ -81,6 +81,9 @@ def public_modules(module):
     + re.escape("The distutils.sysconfig module is deprecated, use sysconfig instead")
     + ":DeprecationWarning",
 )
+@pytest.mark.filterwarnings(
+    "ignore:module 'sre_constants' is deprecated:DeprecationWarning",
+)
```

Comment:
What is responsible for this warning? It would be nice to quickly open an issue for the relevant project, as we did above for Astroid. That way we can track the issue and remove the clutter when the issue is fixed: https://github.com/python-trio/trio/pull/2320

Reply:
It's caused by an old vendored version of pyparsing from `pkg_resources`.

Reply:
I checked and it looks like the other warning filters on that test can be removed.

Reply:
I found the relevant, existing issue in setuptools and I've added a link to that.